set.mm

```diff
@@ -488546,6 +488586,40 @@ such that every prime ideal contains a prime element (this is a
       UULWNUUHUUIUQTWCWHYIWSWLKYJYIWRWJWKWQABWPWJCYPYCWOWESQWFWG $.
   $}
 
+  ${
+    ply1fermltlchr.w $e |- W = ( Poly1 ` F ) $.
+    ply1fermltlchr.x $e |- X = ( var1 ` F ) $.
+    ply1fermltlchr.l $e |- .+ = ( +g ` W ) $.
+    ply1fermltlchr.n $e |- N = ( mulGrp ` W ) $.
+    ply1fermltlchr.t $e |- .^ = ( .g ` N ) $.
+    ply1fermltlchr.c $e |- C = ( algSc ` W ) $.
+    ply1fermltlchr.a $e |- A = ( C ` ( ( ZRHom ` F ) ` E ) ) $.
+    ply1fermltlchr.p $e |- P = ( chr ` F ) $.
+    ply1fermltlchr.f $e |- ( ph -> F e. CRing ) $.
+    ply1fermltlchr.1 $e |- ( ph -> P e. Prime ) $.
+    ply1fermltlchr.2 $e |- ( ph -> E e. ZZ ) $.
+    $( Fermat's little theorem for polynomials in a commutative ring ` F ` of
+       characteristic ` P ` prime, we have the polynomial equation
+       ` ( X + A ) ^ P = ( ( X ^ P ) + A ) ` .  (Contributed by Thierry Arnoux,
```

Comment:
wording? Maybe: "... characteristic ` P ` prime: we have the polynomial equation ... "

It is very helpful to have the formula ` ( X + A ) ^ P = ( ( X ^ P ) + A ) `  here (using the usual exponentiation), because in the theorem the variables are reversed.

Reply:
Wording updated.